src/Controls/samples/Controls.Sample/Resources/Raw/HybridSamplePage/index.html

```diff
@@ -0,0 +1,58 @@
+﻿<!DOCTYPE html>
+
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml">
+<head>
+    <meta charset="utf-8" />
+    <title></title>
+    <script>
+        function SendRawMessageToCSharp() {
```

Comment:
Maybe want to keep the naming more 'dotnet' inspired? `SendRawMessageToDotNet()` ?

Reply:
This is now a proper JS method named `SendRawMessage`.